blocks/inspector-controls/toggle-control/index.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * WordPress dependencies
+ */
+import { withInstanceId } from 'components';
+import Toggle from 'components/form-toggle';
```

Comment:
Minor: We should import from the top-level, i.e. `import { withInstanceId, Toggle } from 'components';`